packages/core/bootstrap/src/lib/server.ts

```diff
@@ -120,23 +114,25 @@ export const initHandler =
       process.exit()
     })
 
+    app.addHook('onClose', async () => {
+      storeSlice('cacheWarmer').dispatch(warmupShutdown())
+      storeSlice('errorBackoff').dispatch(shutdown())
+      storeSlice('ws').dispatch(WSReset())
+      context.cache?.instance?.close()
+    })
+
     return new Promise((resolve) => {
-      const server = app.listen(port, () => {
-        server.on('close', () => {
-          storeSlice('cacheWarmer').dispatch(warmupShutdown())
-          storeSlice('errorBackoff').dispatch(shutdown())
-          storeSlice('ws').dispatch(WSReset())
-          context.cache?.instance?.close()
-        })
-
-        logger.info(`Listening on port ${(server.address() as AddressInfo).port}!`)
-        resolve(server)
+      app.listen(port, '0.0.0.0', function (_, address) {
```

Comment:
Here we specify the IP address, but we don't in `metricsApp`. Is it necessary here? Can we be consistent between the two?

Also, would prefer an arrow function here, for consistency

Reply:
yes, you're right, it should be removed and it's better to use an arrow function :) done!